dev/@bud/sources/command/release.command.ts

```diff
@@ -180,61 +153,89 @@ export class Release extends Command {
    * npm publish
    *
    * @remarks
-   * The code is committed, the tag is pushed, and the
-   * packages are published. Pay attention to the npm tag here.
-   * If this is a next version, then be sure to add the next
-   * tag instead of latest.
+   * The code is committed, the tag is pushed.
+   *
+   * @internal
    */
-  public async publish() {
+  public async push() {
```

Comment:
git and npm tasks are separated into two methods. the problem with settings the auth token etc. is that it will be set in `yarnrc.yml`. So, the automated commit would leak the token.